internal/db/db.go

```diff
@@ -27,7 +27,7 @@ func DbConnect() {
 
 	db, err := gorm.Open(postgres.Open(dsn), &gorm.Config{})
 	if err != nil {
-		panic(err)
+		log.Fatal(err)
```

Comment:
@SteveHNH @tahmidefaz Tiny little thing, but should we start replacing these statements with the "logging" package that we have now?

Reply:
Good suggestion! Lmk make those changes here also! :+1:

Reply:
@aleccohan we should only be using the internal logging now :+1: